Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.37352&attempt_id=95d548bd-9854-437d-a2d3-02449dc80bc4)

Attempted: zig

<details><summary>Partial log (click to expand)</summary><p>

```
[ 99%] Building CXX object CMakeFiles/zig.dir/src/errmsg.cpp.o
[ 99%] Building CXX object CMakeFiles/zig.dir/src/translate_c.cpp.o
[100%] Linking CXX executable zig
/nix/store/ijmisi52n5wbwybg5ldysa6af01zy5dl-binutils-2.28.1/bin/ld: cannot find -lxml2
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/zig.dir/build.make:887: zig] Error 1
make[1]: *** [CMakeFiles/Makefile2:74: CMakeFiles/zig.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
builder for '/nix/store/kf48bxi18gazjs42iavs8lf2nb509smc-zig-0.2.0.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/kf48bxi18gazjs42iavs8lf2nb509smc-zig-0.2.0.drv' failed
```
</p></details>

--- a/pkgs/development/compilers/zig/default.nix
+++ b/pkgs/development/compilers/zig/default.nix
@@ -1,17 +1,18 @@
-{ stdenv, fetchFromGitHub, cmake, llvmPackages }:
+{ stdenv, fetchFromGitHub, cmake, llvmPackages, libxml2 }:
 
 stdenv.mkDerivation rec {
-  version = "0.1.1";
+  version = "0.2.0";
   name = "zig-${version}";
 
   src = fetchFromGitHub {
     owner = "zig-lang";
     repo = "zig";
     rev = "${version}";
-    sha256 = "01yqjyi25f99bfmxxwyh45k7j84z0zg7n9jl8gg0draf96mzdh06";
+    sha256 = "0lym28z9mj6hfiq78x1fsd8y89h8xyfc1jgqyazi1g9r72427n07";
   };
 
-  buildInputs = [ cmake llvmPackages.clang-unwrapped llvmPackages.llvm ];
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ llvmPackages.clang-unwrapped llvmPackages.llvm libxml2 ];
 
   cmakeFlags = [
     "-DZIG_LIBC_INCLUDE_DIR=${stdenv.cc.libc_dev}/include"
@@ -26,7 +27,7 @@ stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "Programming languaged designed for robustness, optimality, and clarity";
-    homepage = http://ziglang.org/;
+    homepage = https://ziglang.org/;
     license = licenses.mit;
     platforms = platforms.unix;
     maintainers = [ maintainers.andrewrk ];

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -11851,7 +11851,9 @@ with pkgs;
     czmq = czmq3;
   };
 
-  zig = callPackage ../development/compilers/zig {};
+  zig = callPackage ../development/compilers/zig {
+    llvmPackages = llvmPackages_6;
+  };
 
   zimlib = callPackage ../development/libraries/zimlib { };
 

